### PR TITLE
Test not-empty sections

### PR DESCRIPTION
**Pull request by avdata99**
Related to [Multi#459](https://github.com/GSA/datagov-ckan-multi/issues/459)

This PR:
 - Add test case for empty sections
 - Fix CKAN 2.3 test environment from this new test
 - Fix docker environment test to use the right _ini_ file

### CODE REVIEW

**Review by jbrown-xentity — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
           command: |
             docker-compose logs db
             docker-compose logs ckan
-            docker-compose exec ckan /bin/bash -c "nosetests --ckan --with-pylons=src_extensions/datagovtheme/docker_test.ini src_extensions/datagovtheme/"
+            docker-compose exec ckan /bin/bash -c "cd src/ckan/ && wget https://raw.githubusercontent.com/GSA/catalog.data.gov/master/tools/ci-scripts/test-catalog-next.ini && nosetests --ckan --with-pylons=test-catalog-next.ini ../../src_extensions/datagovtheme/"
```
Comment (adborden) [inline .circleci/config.yml:36]: Can the wget be done at container build time, instead of test time?

Comment (avdata99) [reply .circleci/config.yml:36]: Yes, I agree. I'm looking the shortest path to achieve this since this file is outside the docker build context in catalog-next repo. I will move it and update extensions

Comment (avdata99) [reply .circleci/config.yml:36]: I created a PR for catalog-next repo @adborden: https://github.com/GSA/catalog.data.gov/pull/156
And 3 PR for extensions to fit this new location
